packages/pwa/app/pages/account/index.jsx

```diff
@@ -88,8 +86,7 @@ const Account = () => {
     // Using Redirect allows us to store the directed page to location
     // so we can direct users back after they are successfully log in
     if (customer.authType != null && !customer.isRegistered) {
-        const path = buildPathWithUrlConfig('/login', {locale})
-        return <Redirect to={{pathname: path, state: {directedFrom: location.pathname}}} />
+        navigate('/login', 'push', {directedFrom: location.pathname})
```

Comment:
One reason I saw is that instead of importing `buildPathWithUrlConfig` to rebuild the path, we can simply call `navigate` hook with a path. It is already aware of site and locale, and we can avoid the hassles of importing the locale and site into this page.

Reply:
As you  pointed out it is not a good practise, I will switch back to use `Redirect` component